vault/mount.go

```diff
@@ -72,6 +72,13 @@ const (
 	MountTableNoUpdateStorage = false
 )
 
+// DeprecationStatus errors
+var (
+	errMountDeprecated     = errors.New("mount entry associated with deprecated builtin")
```

Comment:
Usually we only define err globals like this if we're going to try to match them against errors, e.g. in tests or client code.  What does it buy us over defining these in the func where they're used?